### DISCUSSION TITLE
Convert .raw to monochrome png/jpg

### DISCUSSION THREAD

**Discussion by s0564896** [ImageMagick/ImageMagick#2663] [Help]
I do have a bitmap that I try to interpret. My current approach is to write all information in a .raw file. Now I want to convert it to a monochrome png or jpg (i don't need a specific format). 
How can I do this in the command line? I need to perform this action for much more images.

**Comment by snibgo** [comment]
What version of IM, on what platform?

What data is in your .raw file? If it is just one 16-bit integer per pixel, with no header data, and you know the width and height, you can convert it like this:
```
magick -size 123x234 -depth 16 gray:in.raw out.png
```

**Reply by s0564896** [reply]
Im working on Windows 10, current version of IM
My .raw file contains 1 bit per pixel (monochrome). 
It contains 96 byte of data.
So the size could be anything (I would check maybe something like 12x8)
I guess this would mean something like 
`magick -size 12x8 -depth 1 gray:in.raw out.png`

**Comment by snibgo** [comment]
Yes, that's right.
